src/main/java/org/quartzpowered/network/server/NetworkServer.java

```diff
@@ -42,7 +42,8 @@
 import static io.netty.channel.ChannelOption.TCP_NODELAY;
 
 public class NetworkServer {
-    @Inject private Logger logger;
+    @Inject
+    private Logger logger;
 
```

Comment:
These are supposed to be on one line, please allow one line annotations in your formatting options and update the PR.